src/background/broken-page-report.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import getBrowserInfo from '/utils/browser-info.js';
+import { SUPPORT_PAGE_URL } from '/utils/urls.js';
+
+chrome.runtime.onMessage.addListener((msg, sender, sendResponse) => {
+  if (msg.action === 'report-broken-page') {
+    (async () => {
+      try {
+        const formData = new FormData();
+        const browserInfo = await getBrowserInfo();
+
+        formData.append('support_ticket[user_name]', '');
+        formData.append('support_ticket[user_email]', msg.email);
+        formData.append(
+          'support_ticket[subject]',
+          `[GBE] Broken page report: ${msg.url}`,
+        );
+        formData.append('support_ticket[message]', msg.description);
+        formData.append('support_ticket[selected_browser]', browserInfo.name);
+        formData.append('support_ticket[browser_version]', browserInfo.version);
+        formData.append('support_ticket[selected_os]', browserInfo.osVersion);
+        formData.append('support_ticket[os_version]', '');
+
+        if (msg.screenshot) {
+          const screenshot = await chrome.tabs.captureVisibleTab(null, {
+            format: 'jpeg',
+            quality: 100,
```

Comment:
currently the height defaults to the view port height, can we pass a height of the full page to capture everything?

Reply:
The `rect` option is only supported in Firefox (https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/extensionTypes/ImageDetails#rect), and I think it might be tricky to detect the height of the whole page, and still I am not sure if without manual "scrolling" you can take a screenshot of the part, which is not visible. I would keep it as it is for now.